src/checks.rs

```diff
@@ -34,6 +35,10 @@ trait Check {
     fn run(&self, line: &LineEntry) -> Option<Warning>;
 }
 
+trait GlobalCheck {
```

Comment:
I needed to add this because the type signature is different from the other checks which only needs to know the single lines.